shared/team-building/user-result.tsx

```diff
@@ -0,0 +1,352 @@
+      {keybaseUsernameComponent}
+      {props.isPreExistingTeamMember ? (
+        <Kb.Text type="BodySmall" lineClamp={1}>
+          {isPreExistingTeamMemberText(props.prettyName)}
+        </Kb.Text>
+      ) : (
+        <>
+          <FormatPrettyName
+            displayLabel={props.displayLabel}
+            prettyName={props.prettyName}
+            services={serviceMapToArray(props.services)}
+            showServicesIcons={showServicesIcons}
+          />
+          {/* When the service result does not have any information other than
+            the service username we don't want to show the service icons since
+            there will only be one item */}
+          {showServicesIcons ? (
+            <ServicesIcons
+              services={serviceMapToArray(props.services)}
+              isKeybaseResult={props.isKeybaseResult}
+              prettyName={props.prettyName}
+              displayLabel={props.displayLabel}
+              keybaseUsername={props.keybaseUsername}
+            />
+          ) : null}
+        </>
+      )}
+    </Kb.Box2>
+  )
+}
+
+const Username = (props: {
+  followingState: Types.FollowingState
+  isKeybaseResult: boolean
+  keybaseUsername: string | null
+  username: string
+}) => (
+  <Kb.Text
+    type="BodySemibold"
+    style={followingStateToStyle(
+      props.isKeybaseResult && props.keybaseUsername ? props.followingState : 'NoState'
+    )}
+  >
+    {props.username}
+  </Kb.Text>
+)
+
+const ActionButton = (props: {inTeam: boolean; onAdd: () => void; onRemove: () => void}) => {
+  const Icon = props.inTeam ? AlreadyAddedIconButton : AddButton
+
+  return (
+    <Kb.ClickableBox onClick={props.inTeam ? props.onRemove : props.onAdd}>
+      <Kb.Box2
+        className="actionButton"
+        direction="vertical"
+        centerChildren={true}
+        style={Styles.collapseStyles([styles.actionButton, props.inTeam && {backgroundColor: null}])}
+      >
+        <Icon />
+      </Kb.Box2>
+    </Kb.ClickableBox>
+  )
+}
+
+const AddButton = () => <Kb.Icon type="iconfont-circle" fontSize={22} color={Styles.globalColors.black_10} />
+
+const AlreadyAddedIconButton = () => (
+  <Kb.Icon type="iconfont-success" fontSize={22} color={Styles.globalColors.blue} />
+)
+
+const ActionButtonSize = 40
+export const userResultHeight = Styles.globalMargins.xlarge
+const styles = Styles.styleSheetCreate(() => ({
+  actionButton: Styles.platformStyles({
+    common: {
+      height: ActionButtonSize,
+      marginLeft: Styles.globalMargins.tiny,
+      width: ActionButtonSize,
+    },
+    isMobile: {
+      marginRight: Styles.globalMargins.small,
+    },
+  }),
+  addToTeamIcon: {
+    ...Styles.globalStyles.rounded,
+    height: ActionButtonSize,
+    width: ActionButtonSize,
+  },
+  bottomRowContainer: {
+    alignItems: 'baseline',
+    // TODO: The smallest desktop width pushes text incorrectly.
+    // TODO: Long prettyName and services can push underneath the ActionButton.
```

Comment:
iPhone SE and smallest desktop width wrap the `prettyName` in the bottom row. Working on how to handle this with design. But the current behavior is no different than the previous.